docs/products/metrics/concepts/retention-rules.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Retention rules
+---
+Retention periods in Aiven for Metrics® help you manage how long your metrics data is stored. Understanding how to set these periods allows you to optimize storage space while maintaining access to historical data for analysis.
+
+## Understand retention rules
+
+Retention rules define the duration for which Aiven for Metrics stores your metrics data.
```

Comment:
There's a lilttle bit of repeat between this paragraph and the first one.  "help you manage how long your metrics data is stored." is here as "the duration for which Aiven for Metrics stores your metrics data". 

Can we remove the repetition? Also do we need the "understand retention rules" title? I see limited seo or readbility benefits, and this probably pushes for repetion, why not just have a couple of paragraphs directly under the main title?

Reply:
Update the cotnent.